controller.rb

```diff
@@ -0,0 +1,41 @@
+get '/' do
+  erb :home
+end
+
+get '/apartments' do
+  @apartments=Apartment.all
+  erb :"apartments/index"
+end
+
+get '/tenants' do
+  @tenants=Tenant.all
+  erb :"tenants/index"
+end
+
+get '/apartments/new' do
+  @apartments=Apartment.all
+  erb :"apartments/new"
+end
+
+get '/apartments/:id' do
+  @id=params[:id].to_i
```

Comment:
ActiveRecord will handle the string to integer conversion so `.to_i` is not needed